fedimint-testing/src/bin/fixtures.rs

```diff
@@ -1,51 +1,53 @@
+use std::collections::HashMap;
 use std::env;
+use std::io::Write;
+use std::path::PathBuf;
 use std::sync::Arc;
 use std::time::Duration;
 
 use bitcoincore_rpc::{Client as BitcoinClient, RpcApi};
+use clap::{Parser, Subcommand};
+use fedimint_client::module::gen::{ClientModuleGenRegistry, DynClientModuleGen};
+use fedimint_core::config::load_from_file;
+use fedimint_core::core::LEGACY_HARDCODED_INSTANCE_ID_WALLET;
+use fedimint_core::db::Database;
 use fedimint_core::task::{sleep, TaskGroup};
+use fedimint_ln_client::LightningClientGen;
 use fedimint_logging::TracingSetup;
-use tokio::process::Command;
-use tokio::sync::Mutex;
+use fedimint_wallet_client::config::WalletClientConfig;
+use fedimint_wallet_client::WalletClientGen;
+use mint_client::modules::mint::MintClientGen;
+use mint_client::{module_decode_stubs, Client, UserClient, UserClientConfig};
+use tokio::fs::{self, OpenOptions};
+use tokio::io::AsyncWriteExt;
+use tokio::process::{Child, Command};
+use tokio::sync::mpsc::{self, Receiver, Sender};
 use tracing::{error, info, trace};
 use url::Url;
 
-#[derive(Default, Clone)]
-struct State {
-    bitcoin_rpc: bool,
+#[derive(Subcommand)]
+enum Cmd {
+    // daemons
+    Bitcoind,
+    Lightningd,
+    Lnd,
+    Electrs,
+    Esplora,
+    Daemons,
```

Comment:
Maybe rename this to AllDaemons? I needed to look at the code to determine which daemons this was referring to